docs/diagrams/LessonAddSequenceDiagram.puml

```diff
@@ -0,0 +1,92 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":LessonAddCommandParser" as LessonAddCommandParser LOGIC_COLOR
+participant "la:LessonAddCommand" as LessonAddCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+participant "editedPerson:Person" as Person LOGIC_COLOR
+participant "lesson:Lesson" as Lesson LOGIC_COLOR
+
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("ladd 1 date/20 Jan 2021 time/1300-1500 subject/Science hw/Exercise 2")
```

Comment:
Inputs for diagrams can be shorter, currently the inputs are too long and it wastes a lot of space causing the actual important parts to be too small